api/specs/webserver/openapi-projects.yaml

```diff
@@ -222,6 +217,31 @@ paths:
         default:
           $ref: "#/components/responses/DefaultErrorResponse"
 
+  /projects/{project_id}/duplicate:
```

Comment:
:duplicate maybe?

Reply:
no, in this file all the routes are like this.